test/lib/nockselements.js

```diff
@@ -183,7 +183,7 @@ module.exports = {
   elementsByTag(selector, foundArray) {
     this._addNock(this._requestUri)
       .persist()
-      .post(this._protocolUri + 'elements', {'using': 'tag name', 'value': selector || 'nock'})
+      .post(this._protocolUri + 'elements', {'using': 'css selector', 'value': selector || 'nock'})
```

Comment:
this should be `tage name`m as we are testing `elementsByTag`

Reply:
`localteStrategy:tagName` is replaced by `css selector` inside selenium so we need mock server for css selector instead of tag name